android/assets/jsons/translations/Dutch.properties

```diff
@@ -967,20 +961,17 @@ Monument = Monument
 Granary = Graanschuur
 
 Must not be on [terrain] = Mag niet op [terrain] zijn
- # Requires translation!
-Stone Works = 
+Stone Works = Steengroeve
 
 'Time crumbles things; everything grows old and is forgotten under the power of time' - Aristotle = '
 Stonehenge = Stonehenge
 
 [stats] Per [amount] Population in this city = [stats] Per [amount] Bevolking in deze stad
 Library = Bibliotheek
 
- # Requires translation!
-'Libraries are as the shrine where all the relics of the ancient saints, full of true virtue, and all that without delusion or imposture are preserved and reposed.' - Sir Francis Bacon = 
+'Libraries are as the shrine where all the relics of the ancient saints, full of true virtue, and all that without delusion or imposture are preserved and reposed.' - Sir Francis Bacon = 'Bibliotheken zijn als het heiligdom waar alle relikwieën van de oude heiligen, vol van ware deugd, en dat alles zonder misleiding of bedrog worden bewaard en bewaard.' - Sir Francis Bacon
```

Comment:
Deze zin loopt niet, maar ik geef toe dat die dat ook niet doet in het Engels. 'bewaard en bewaard' klinkt wat vaag, misschien eerder 'bewaard en beschermt' of zoiets?